tests/jballerina-unit-test/src/test/resources/test-src/expressions/invocations/function-stmt.bal

```diff
@@ -1,3 +1,21 @@
+// Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Licence header should be 2022

Reply:
This file had been added in 2019 without the header. So I added the header mentioning the year as 2019.